### ISSUE TITLE
 cur_input_embeds = torch.cat([cur_input_embeds_1, cur_image_features[0:0], cur_input_embeds_2], dim=0),其中cur_image_features[0:0]表示这是一个没有维度的向量，图像的特征并没有真正加进去

### ISSUE THREAD

**Issue by hangzeli05** [X-PLUG/mPLUG-Owl#193]
mPLUG-Owl2中的代码错误

**Comment by vateye** [comment]
No, it is for compatible with deepspeed zero3 during training on text samples. For multi-modal input, this would not encounter.
